.github/workflows/acceptance-tests.yml

```diff
@@ -47,7 +47,6 @@ jobs:
           generic:
             - 'mongodbatlas/data_source_mongodbatlas_backup_compliance_policy*.go'
             - 'mongodbatlas/resource_mongodbatlas_backup_compliance_policy*.go'
-            - 'mongodbatlas/data_source_mongodbatlas_data_lakes_test*.go'
```

Comment:
I see you removed also private endpoint adl. Do we also have to remove it from here?

Reply:
In this file we have:

```
          network:
            - 'mongodbatlas/**_network**.go'  
            - 'mongodbatlas/**_privatelink**.go' 
            - 'mongodbatlas/**_private_endpoint**.go'
```

which cannot be removed because we still have:
```
terraform-provider-mongodbatlas/mongodbatlas/resource_mongodbatlas_privatelink_endpoint_serverless.go
```

I think there is no other change needed for this file @andreaangiolillo

Reply:
Thanks for checking!